_posts/2023-12-06-#28690.md

```diff
@@ -7,21 +7,45 @@ authors: [TheCharlatan]
 components: ["build system"]
 host: stickies-v
 status: upcoming
-commit:
+commit: de6d79847537cd7d0dfc3e2dd7a57b39b4281b5a
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- The [libbitcoinkernel project](https://github.com/bitcoin/bitcoin/issues/27587) is an effort to decouple Bitcoin Core’s consensus engine from other non-consensus modules in the codebase. We have previously covered libbitcoinkernel-related PRs [#27711](/27711), [#25527](/25527), [#24410](/24410) and [#20158](/20158). However, they are not essential to understanding this PR as they focused more on internal code reorganization as opposed to this PR's focus on the build system.
+
+- Suggested reading: [Static and dynamic libraries](https://www.learncpp.com/cpp-tutorial/a1-static-and-dynamic-libraries/) (up until the "Installing and using libraries" section which is not relevant).
+
+- Most of the [libraries](https://github.com/bitcoin/bitcoin/blob/master/doc/design/libraries.md) _(required reading)_ that are built are internal and static, and can't be used outside of Bitcoin Core. Examples of internal libraries are `libbitcoin_cli`, `libbitcoin_common` and `libbitcoin_node`. There are only two [external (dynamic, or shared) libraries](https://github.com/bitcoin/bitcoin/blob/master/doc/shared-libraries.md) _(required reading)_: `libbitcoinconsensus` and `libbitcoinkernel`, even though the latter is not currently documented as such since the API is not ready.
+
+- This PR introduces a new _internal_ `libbitcoin_kernel` library, in addition to the existing external `libbitoincoinkernel` library.
 
+- Internal libraries are statically built to avoid dependency on external code (i.e. the shared library) at runtime, which could be an attack vector.
+
+- An explanation of some abbreviations used:
+  - `a` vs `la`(as in `libbitcoin_kernel_a_SOURCES` and `libbitcoinkernel_la_SOURCES`):
+    - `a` stands for `archive` (i.e. a static library)
+    - `la` stands for [`Libtool Archive`](https://www.linuxfromscratch.org/blfs/view/svn/introduction/la-files.html)
 
 ## Questions
 
 1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)? What was your review approach?
```

Comment:
For the meeting, it might also be helpful to ask if people read libraries.md and the "Static and Dynamic Libraries" readings.